working/augmentation-libraries/feature-specification.md

```diff
@@ -650,10 +654,11 @@ More specifically:
     usual, external and abstract variables cannot augment their
     initializing expression, since it does not exist.*
 
-    Augmenting initializer expressions replace the augmented initializer. The
-    augmenting initializer may use an `augmented` expression which executes the
-    augmented initializer expression when evaluated. If no initializer is
-    provided then the augmented initializer is not altered.
+    Augmenting initializer expressions replace the augmented initializer (or
+    provide one where none existed previously). The augmenting initializer
+    may use an `augmented` expression which executes the augmented initializer
```

Comment:
```suggestion
    may use an `augmented` expression which evaluates the augmented initializer
```
The word 'evaluates' seems more informative than 'executes' in the case where the value of the expression is typically the only outcome of this step, and certainly an important outcome in any case.